.github/workflows/canary-integration-test.yml

```diff
@@ -1323,7 +1310,7 @@ jobs:
         uses: ./.github/workflows/tmate_debug
         with:
           use-tmate: ${{ secrets.USE_TMATE }}
-          limit-access-to-actor: true
+
```

Comment:
Multiple blanks lines

Reply:
will revert on pr is verified by someone other then me